configure

```diff
@@ -3099,12 +3102,26 @@ EOF
 $as_echo "$OCAML_OS_TYPE" >&6; }
 
 
-if test "${OCAML_OS_TYPE}" = "Win32" ; then
+if  test "${OCAML_OS_TYPE}" = "Win32"; then :
```

Comment:
Out of curiosity, is this `:` at the end for portability on Windows?

Reply:
That's autoconf, not me :smile: That's coming from `AS_IF` (I try to follow m4sh coding standards, although m4sh, coding and standards I find to be co-oxymoronic terms!). I think it's a cheap way of ensuring that you never get an empty `then` branch, as that would be a syntax error. There's similar things in the `Makefile`s of the OCaml testsuite, IIRC.